cypress.json

```diff
@@ -9,5 +9,11 @@
   "chromeWebSecurity": false,
   "projectId": "xqvxh7",
   "video": true,
-  "videoUploadOnPasses": false
+  "videoUploadOnPasses": false,
+  "e2e": {
+    "baseUrl": "http://localhost:3000"
+  },
+  "env": {
+    "council_website_homepage_url": "https://www.newark-sherwooddc.gov.uk"
```

Comment:
No, this is passed to the cypress test. This was more to make the tests more configurable. In the cypress tests, you can refer to the environment variable rather than its plain text counterpart if wanted.
For example, checking the [contact us link](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/33/files#diff-55a756869d02142a0794936ec414ed8ae6ab86f1d7c843e45fc68dc2fe427a13R25) and checking the [privacy notice](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/33/files#diff-55a756869d02142a0794936ec414ed8ae6ab86f1d7c843e45fc68dc2fe427a13R64)